### PR TITLE
Remove RTT0 and RTT1 support

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Removes the support for RTT0 and RTT1 as we don't have a port to any
hardware with RTT0 and RTT1.

Signed-off-by: Jens Wiklander <jens.wiklander@linaro.org>

### CODE REVIEW

File: core/arch/arm/include/mm/tee_mm_def.h [hunk 1]
```diff
@@ -32,37 +32,4 @@
 #define SMALL_PAGE_MASK		0x00000fff
 #define SMALL_PAGE_SIZE		0x00001000
 
-/*
```
Comment (pascal-brand-st-dev) [inline core/arch/arm/include/mm/tee_mm_def.h]: Is it possible to move `SMALL_PAGE_xxx` macros somewhere else to remove `tee_mm_def.h`?

**Comment by pascal-brand-st-dev** [comment]
Would be good to link this PR to #291 which becomes non-relevant.

Apart from a comment on macros `SMALL_PAGE_xxx`,

    Reviewed-by: Pascal Brand <pascal.brand@linaro.org>

File: core/arch/arm/include/mm/tee_mm_def.h (hunk 1 above)
Comment (jenswi-linaro) [inline core/arch/arm/include/mm/tee_mm_def.h]: I have that in another patch that I intend to upload after this PR, but I can add it here if you prefer that.

Comment (pascal-brand-st-dev) [inline core/arch/arm/include/mm/tee_mm_def.h]: In that case, it is fine. Do not change anything.

**Comment by jenswi-linaro** [comment]
Review tag applied

**Pull request merged**